src/app/docs/page/page-docs.component.html

```diff
@@ -0,0 +1,31 @@
+<sky-docs-demo-page
+  moduleName="SkyPageModule"
+  moduleSourceCodePath="src/app/public/modules/page/"
+  pageTitle="Page"
+>
+  <sky-docs-demo-page-summary>
+    The page component wraps the contents of an index.html file....
+  </sky-docs-demo-page-summary>
+
+  <sky-docs-demo>
+    <div
+      class="sky-text-info sky-margin-stacked-content-section"
+    >
+      We are creating a page component demo for this section.
+    </div>
+  </sky-docs-demo>
+
+  <!-- <sky-docs-code-examples
+    [packageDependencies]="{
+      '@skyux/lists': '*'
+    }"
+  >
+    <sky-docs-code-example
+      sourceCodePath="src/app/public/plugin-resources/code-examples/page"
+    >
+    </sky-docs-code-example>
+    <sky-docs-code-example
+      sourceCodePath="src/app/public/plugin-resources/code-examples/page`"
+    >
+    </sky-docs-code-example> -->
+</sky-docs-demo-page>
```

Comment:
We should slap an `@internal` tag on the `SkyPageThemeAdapterService`. Those aren't to be used by consumers.

Reply:
Will do.